src/pages/AddPhoto/AddPhoto.tsx

```diff
@@ -11,7 +11,16 @@ interface AddPhotoProps {
 }
 
 export const AddPhoto: React.FC<AddPhotoProps> = ({ setUploadedFileData, uploadedFileData }) => {
-    const isVisible = uploadedFileData !== "";
+    const [isVisible, setIsVisible] = useState(false);
+
+    // Reset file data to empty string if user uses back button
+    useEffect(() => {
+        setUploadedFileData("");
+    }, [setUploadedFileData]);
```

Comment:
That's cool, good idea. I notices a (possibly totally unrelated) bug with using the back button: the tool icons disappear (straight line vs curvy line)

Reply:
![image](https://user-images.githubusercontent.com/34172843/103313558-40dcb580-49d5-11eb-8475-13eff65b12be.png)

Reply:
After looking at it some more, I don't think we should have the back button enabled in the app. There are too many places our workflow is only supposed to move forward - for example, the page where you review the path you just drew.

Reply:
I am not able to reproduce this bug. Which page did you go back from?

Reply:
after pressing 'keep', press 'back' twice.

Reply:
but I guess if we don't support the back button, we can leave it...

Reply:
That would be for a different pr, but I searched a little and there seem to be ways to prevent the user from using the back button. This way we would be consistent with the workflow of the app, while still preventing users from seeing weird behavior if they happen to press back (which users usually end up trying imo... ) https://medium.com/@subwaymatch/disabling-back-button-in-react-with-react-router-v5-34bb316c99d7

Reply:
I'll do this in a subsequent pr.